lldb/include/lldb/Symbol/CompilerType.h

```diff
@@ -194,6 +192,54 @@ class CompilerType {
   bool IsTypedefType() const;
 
   bool IsVoidType() const;
+
+  bool IsSmartPtrType() const;
```

Comment:
Some methods like this one could benefit from a Doxygen comment that explains what the semantics are.

Reply:
Done.